exercises/takudzwa_makoni/ex02_oo_basics/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required ( VERSION 2.8.5 )
+
+###############################################################################
+# Setup
+###############################################################################
+# Name
+project ( Shapes )
+
+# Find custom cmake modules
+set ( CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../CMake")
+
+# Common setup
+include ( CommonSetup )
+
+###############################################################################
+# Executable
+###############################################################################
+
+set ( SRC_FILES
+  src/main.cpp
+  # add any additional source files here
+)
+
+set ( HDR_FILES
+  # add any additional header files here
+  src/shape.h
+  src/square.h
+  src/rectangle.h
+  src/isoTriangle.h
+  src/circle.h
+  src/shapeSorter.h
+
```

Comment:
I like the matching now between physical and logical units in your codebase 👍 

Note that you should put implementations in `cpp` files.  Please attempt this for say `class circle`, keep declarations in the header, but move the definitions into a new `src/circle.cpp`.

Reply:
will add cpp files for the classes👍

Reply:
Just doing one is fine.